distro/assembly/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM jboss/wildfly
+
+RUN $JBOSS_HOME/bin/add-user.sh admin artificer1! --silent
```

Comment:
@brmeyer this is an admin user to JBoss Wildfly console. As the artificer user is not an admin user in the ManagementRealm, this user will be needed to access the console, just in case something needs to be done through it.
This is a development image.
In an official image, there will not be a management user provided by default, as per the guidelines qe have.

Cheers,